fastats/core/single_pass.py

```diff
@@ -9,7 +9,7 @@ def value(x):
 
 
 @fs
-def single_pass(x):
+def single_pass(x):  # pragma: no cover
```

Comment:
We need to work out a way of avoiding the JIT on as-needed basis, to test with both JIT and non JIT. For now, I think we should keep these as `no cover`.

Reply:
I agree that we need a proper solution to jit-vs-nonjit, but i totally disagree with `pragma: no cover` in an `@fs` function.

I think we need to maintain python and numba tests to ensure complete coverage until we have a better solution

Reply:
That's OK, but the target coverage won't be attainable, making builds fail.

Reply:
Yea, we'll take it on a case-by-case basis, but so far it's always been possible to get the coverage using pure python functions, but it's definitely slightly fiddly and a bit annoying.

I'd still prefer 'a bit annoying' for developers over 'not covered' though 😄